src/ng-generate/table/index.ts

```diff
@@ -37,6 +37,7 @@ import {TsComponentGenerator} from './generators/ts-component.generator';
 import {addModuleImportToModule} from '@angular/cdk/schematics';
 import ora from 'ora';
 import {WIZARD_CONFIG_FILE} from '../table-prompter/index';
+import {ViewEncapsulation} from '@schematics/angular/component/schema';
```

Comment:
Looks like it is unnecessary import here

Reply:
yep

Reply:
done